interpreter/package.go

```diff
@@ -26,16 +29,17 @@ type Package struct {
 	sideEffects []SideEffect
 }
 
-func NewPackageWithValues(name string, obj values.Object) *Package {
+func NewPackageWithValues(name, path string, obj values.Object) *Package {
 	return &Package{
 		name:   name,
+		path:   path,
 		object: obj,
 	}
 }
 
 func NewPackage(name string) *Package {
 	obj := values.NewObject(semantic.NewObjectType(nil))
-	return NewPackageWithValues(name, obj)
+	return NewPackageWithValues(name, "", obj)
```

Comment:
Is an empty path like this the current directory?

Reply:
It means we don't know the path, but this function is mostly obsolete and doesn't work anymore. It's used in some tests where we don't run into this issue with the path.